docs/configuration.adoc

```diff
@@ -258,6 +258,8 @@ use_encryption = true
 kms_key_id = <kms_key_id>
 enable_pv_encryption_in_transit = false
 node_pool_volume_kms_key_id = <node_pool_volume_kms_key_id>
+enable_operator_pv_encryption_in_transit = false
+operator_volume_kms_id = <operator_volume_kms_id>
```

Comment:
You need to add subsections here so you can distinguish between bastion, operator and oke's respective KMS integration. 

Right now, this section tells the user what parameters they need in order to do this integration for OKE **only**. Since you're adding similar capability for bastion and operator, add subsections for them